Makefile

```diff
@@ -120,19 +120,46 @@ letsencrypt:
 	&& . ./devops/scripts/load_env.sh ./templates/core/tre.env \
 	&& ./templates/core/terraform/scripts/letsencrypt.sh
 
-tre-stop:
-	echo -e "\n\e[34m»»» 🧩 \e[96mStopping TRE\e[0m..." \
+tre-start:
+	echo -e "\n\e[34m»»» 🧩 \e[96mStarting TRE\e[0m..." \
 	&& . ./devops/scripts/check_dependencies.sh azfirewall \
 	&& . ./devops/scripts/load_env.sh ./templates/core/.env \
 	&& . ./devops/scripts/load_env.sh ./devops/.env \
-	&& ./devops/scripts/control_tre.sh stop
+	&& ./devops/scripts/control_tre.sh start
 
-tre-start:
-	echo -e "\n\e[34m»»» 🧩 \e[96mStarting TRE\e[0m..." \
+tre-stop:
+	echo -e "\n\e[34m»»» 🧩 \e[96mStopping TRE\e[0m..." \
 	&& . ./devops/scripts/check_dependencies.sh azfirewall \
 	&& . ./devops/scripts/load_env.sh ./templates/core/.env \
 	&& . ./devops/scripts/load_env.sh ./devops/.env \
-	&& ./devops/scripts/control_tre.sh start
+	&& ./devops/scripts/control_tre.sh stop
+
+firewall-install:
```

Comment:
Discussion point: 
I haven't yet included these commands as part of the main deploy. Should we include these services by default?

Reply:
Yes, for local development we will need to create a Service Principal. As far as I understand, there is already one for running in CI.

Reply:
I vote we get on with adding shared services via the API rather than spend too much time smoothing out this process

Reply:
@damoodamoo  agree.

Reply:
I was just going to add a little README on how to create a service principal for deployment for the local dev flow and be done with it for now

Reply:
@tanya-borisova where are these new bundles being deployed as part of the pipeline via `make all`? I can't see it in the PR. They do need to be deployed - especially when others pick up the repository given templates are dependent.

Reply:
@marrobi  done! I added them as a step to tre-deploy, and there is a script that creates a service principal too, that hopefully will make it easier for people